packages/core/src/browser/shell/application-shell.ts

```diff
@@ -392,6 +393,35 @@ export class ApplicationShell extends Widget {
         }
     }
 
+    /**
+     * Reveal a widget in the application shell. This makes the widget visible,
+     * but does not assign focus to it.
```

Comment:
should we say `does not active it`, instead.?

Reply:
Yes, I'll clarify the comments.